### PR TITLE
Update tidb-profile

### PR DESCRIPTION
**Pull request by shiyuhang0**
## Description & motivation
dbt-tidb v1.2.0 has been published, and the corresponding doc needs to be updated.
https://github.com/pingcap/dbt-tidb/pull/6

- Add a new config in the profile
- Add grant and connection retry in feature support 
- Fix inaccurate feature support

### CODE REVIEW

**Comment by shiyuhang0** [comment]
@zhangyangyu @dataders PTAL

File: website/docs/reference/warehouse-profiles/tidb-profile.md [hunk 1]
```diff
@@ -43,20 +43,24 @@ dbt-tidb:
       schema: database_name
       username: tidb_username
       password: tidb_password
+
```
Comment (zhangyangyu) [inline website/docs/reference/warehouse-profiles/tidb-profile.md:46]:
```suggestion
```

**Comment by zhangyangyu** [comment]
Add yourself to the author plz :-)

**Review by zhangyangyu — APPROVED** [review]

File: website/docs/reference/warehouse-profiles/tidb-profile.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ If you're interested in contributing, check out the source code repository liste
 ## Overview of dbt-tidb
 
 **Maintained by:** PingCAP      
-**Author:** Xiang Zhang and Qiang Wu  
+**Author:** Xiang Zhang, Qiang Wu, Yuhang Shi
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/tidb-profile.md:15]: might be worth adding this line now that we have it?
```suggestion
**Author:** Xiang Zhang, Qiang Wu, Yuhang Shi
**dbt Slack Channel**: [#db-tidb](https://getdbt.slack.com/archives/C03CC86R1NY)
```

**Review by dataders — APPROVED** [review]
looks great -- cool to see so many added features in this release!

**Pull request merged**